package.json

```
@@ -14,6 +14,7 @@
     "test:watch": "vitest watch --coverage --no-isolate",
     "test:silent": "vitest run --silent --no-isolate",
     "test:create": "node scripts/create-test/create-test.js",
+    "eggMove:parse": "node scripts/parse-egg-moves/main.js",
```

```suggestion
    "eggMoves:parse": "node scripts/parse-egg-moves/main.js",
```
Should probably be `eggMoves` (I notice it's inconsistently referred to as `eggMove:parse` and `eggMoves:parse` in comments/text throughout the other files).